src/main/kotlin/no/nav/syfo/infrastructure/pdl/PdlClient.kt

```diff
@@ -0,0 +1,188 @@
+package no.nav.syfo.infrastructure.pdl
+
+import io.ktor.client.*
+import io.ktor.client.call.*
+import io.ktor.client.request.*
+import io.ktor.client.statement.*
+import io.ktor.http.*
+import io.micrometer.core.instrument.Counter
+import no.nav.syfo.api.ClientEnvironment
+import no.nav.syfo.api.httpClientDefault
+import no.nav.syfo.domain.PersonIdent
+import no.nav.syfo.infrastructure.NAV_CALL_ID_HEADER
+import no.nav.syfo.infrastructure.azuread.AzureAdClient
+import no.nav.syfo.infrastructure.azuread.AzureAdToken
+import no.nav.syfo.infrastructure.bearerHeader
+import no.nav.syfo.infrastructure.metric.METRICS_NS
+import no.nav.syfo.infrastructure.metric.METRICS_REGISTRY
+import no.nav.syfo.infrastructure.pdl.dto.*
+import no.nav.syfo.infrastructure.redis.RedisStore
+import org.slf4j.LoggerFactory
+
+class PdlClient(
+    private val azureAdClient: AzureAdClient,
+    private val pdlEnvironment: ClientEnvironment,
+    private val cache: RedisStore,
+) {
+
+    suspend fun getPdlIdenter(
```

Comment:
Tror denne kun er brukt i forbindelse med identhendelser. Så mulig denne koden kommer til å ligge død helt til vi innfører det 🤔 Med mindre dere ser noen use-caser for denne som ikke handler om identhendelser?

Reply:
Tipper denne kan fjernes (enn så lenge i hvert fall)